build.gradle

```diff
@@ -114,6 +114,10 @@ subprojects {
 
       spotless {
         java {
+          if (project.path.equals(':server')) {
```

Comment:
Would be better to just add this to `server/build.gradle`

Reply:
I didn't think that would work, but turns out it does, which is much tidier, thank you. Unfortunately, I then had to wrap the config in a try/catch because unless a project opts-in to formatting, the `spotless()` method doesn't exist. I can removed the extra cruft once we enable formatting.